packages/preview/typst-railroad/0.1.0/typst.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "typst-railroad"
```

Comment:
Package names should not include "Typst", and should not be the most obvious/canonical name for what the package does. Can you please propose another name?

Reply:
Sorry, but this is still too descriptive of what the package does. Can you add a prefix/suffix, or an adjective that makes the name more unique, or make a pun with "railroad"/"train"/etc. please?

Reply:
Can I use the name translated to esperanto?
`fervojo`

Reply:
Yes, that would work.